### DISCUSSION TITLE
Feature Request: Product "Tour" / "Onboarding" similar to ngx-ui-tour

### DISCUSSION THREAD

**Discussion by cheezist** [primefaces/community#957] [PrimeNG] [answered]
Hello,

This is a request for a **Tour** or **Onboarding** feature similar to those found on sites that, on initial visit, walks a user through the features of a site.  This would be similar to existing libraries like [ngx-ui-tour](https://github.com/hakimio/ngx-ui-tour) and others in the React-land outlined on this article: [React Onboarding Tour](https://userguiding.com/blog/react-onboarding-tour/)

This feature has many benefits for developers needing to communicate to their users on how to use the tools they've created.  Personally, I would stay away from something like _ngx-ui-tour_ simply because of all its dependencies.  PrimeNG already has many of the components and styles required to build something like this and with developers making complex UIs it would be a natural fit into PrimeNG.

Note: I did not find any other feature requests similar to this, but the words **Tour** and **Overview** have many synonyms and other generic uses in other contexts, so please forgive me if this has already been requested.

Thank you!

**Comment by cheezist** [comment]
Another library that was mentioned in the PrineNG Discord channel was [driver.js](https://driverjs.com/)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback we'll wait for the community upvotes for it.
